tensorflow-core/pom.xml

```diff
@@ -480,7 +454,7 @@
     </profile>
 
     <!-- Disable platform dependency when building the artifact for the dependency itself -->
-    <profile>
+    <!--profile>
```

Comment:
Why did this profile get commented out?

Reply:
Despite the comment, I'm not sure to understand what this profile is used for but I know that the deployment script fails when it is present but everything works fine when it is not.

Reply:
Maybe we should delete it then. We can always pull it out of the history.

Reply:
Yes, that is what I ended up to do.